libkbfs/folder_branch_ops.go

```diff
@@ -5593,6 +5679,7 @@ func (fbo *folderBranchOps) ClearPrivateFolderMD(ctx context.Context) {
 	}
 
 	fbo.head = ImmutableRootMetadata{}
+	fbo.headStatus = headTrusted
```

Comment:
Wouldn't `headUntrusted` make more sense here?  We basically want to set it back to the default value.